### DISCUSSION TITLE
Problem in extracting words by Font Style (face)

### DISCUSSION THREAD

**Discussion by ProxyAyush** [jsvine/pdfplumber#957] [Q&A]
I cannot extract words in a pdf with using fontname  in objects property, my code is below

`import pdfplumber

def get_filtered_text(file_to_parse: str) -> str:
    with pdfplumber.open(file_to_parse) as pdf: 
        for i in range(0, 286):
         text = pdf.pages[i]
         clean_text = text.filter(lambda obj: obj["object_type"] == "char" and obj["size"] == 33)
         #if clean_text.extract_text() != "":
          #print(clean_text.extract_text())

         clean_text2 = text.filter(lambda obj: obj["object_type"] == "char" and obj["size"] <= 11 and obj["fontname"] == "Arial")
         if clean_text2.extract_text() != "":
          print(clean_text2.extract_text())
          
        

get_filtered_text("/Users/User/Desktop/kundu modified.pdf")`

**Comment by jsvine** [comment]
Hi @ProxyAyush, and thanks for your interest in this library. Without the PDF, this will be difficult to diagnose. Can you provide it?

And are you sure that the `fontname` property of the characters you want is _exactly_ the string "Arial"? And, if so, are you sure that there are such characters with a `size` smaller than 11?

Also, if you could, please edit your message to [fix the formatting issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks), which currently are making it a bit difficult to read the code:

> <img width="619" alt="Screenshot" src="https://github.com/jsvine/pdfplumber/assets/534702/24cc9a3e-e114-432d-8af0-6cd5b4950a5f">
